DEVELOPERS.md

```diff
@@ -53,14 +53,23 @@ This will run the server on the port configured
 You can now go to `http://127.0.0:8001/docs` do examine and try the API.
 
 
-## Generate token
+## Test client
```

Comment:
For total n00bs, would a few words here explaining the use-case for why you created this be helpful? i.e. the situations when you might want to use a Test Client.

Otherwise people might skim over this, not lodge it in their heads, and then miss out on it as a useful thing when they get stuck later on.

Reply:
updated with more documentation.